sqlengine/postgres_engine.go

```diff
@@ -245,7 +245,11 @@ func (d *PostgresEngine) URI(address string, port int64, dbname string, username
 }
 
 func (d *PostgresEngine) JDBCURI(address string, port int64, dbname string, username string, password string) string {
-	return fmt.Sprintf("jdbc:postgresql://%s:%d/%s?user=%s&password=%s", address, port, dbname, username, password)
+	jdbcuri := fmt.Sprintf("jdbc:postgresql://%s:%d/%s?user=%s&password=%s", address, port, dbname, username, password)
+	if d.requireSSL {
+		jdbcuri = jdbcuri + "?ssl=true"
```

Comment:
This isn't quite correct - it should be `&` instead of `?`.

This does make me a bit nervous about how we're constructing this. I think it would be better to construct these params using [`url.Values`](https://golang.org/pkg/net/url/#Values).